pkg/gui/list_context_config.go

```diff
@@ -54,6 +54,25 @@ func (gui *Gui) branchesListContext() *context.BranchesContext {
 	)
 }
 
+func (gui *Gui) worktreesListContext() *context.WorktreesContext {
+	return context.NewWorktreesContext(
+		func() []*models.Worktree { return gui.State.Model.Worktrees },
+		gui.Views.Worktrees,
+		func(startIdx int, length int) [][]string {
+			return slices.Map(gui.State.Model.Worktrees, func(worktree *models.Worktree) []string {
+				return presentation.GetWorktreeDisplayString(
+					gui.helpers.Worktree.IsCurrentWorktree(worktree),
+					gui.helpers.Worktree.IsWorktreePathMissing(worktree),
+					worktree)
+			})
+		},
+		nil,
+		gui.withDiffModeCheck(gui.worktreesRenderToMain),
```

Comment:
no need for the diff mode check: that's more applicable to things like branches/commits